classes/settings_provider.php

```diff
@@ -907,11 +917,11 @@ public static function get_seb_settings_map() : array {
                 'seb_allowuserquitseb' => [
                     'seb_quitpassword' => [],
                 ],
-                'seb_allowedbrowserexamkeys' => [],
```

Comment:
I also removed these, which I think was wrong, yet these are not seb settings.

Reply:
I'd leave it here. As this get_seb_settings_map gives us a good overview of how settings are apply to the form. They are not all SEB settings. See seb_suppresssebdownloadlink . Maybe we should rename the method